packages/currency/src/types.ts

```diff
@@ -1,29 +1,33 @@
 import { RequestLogicTypes } from '@requestnetwork/types';
 
+/** Native Currency types */
+export type NativeCurrencyType = RequestLogicTypes.CURRENCY.BTC | RequestLogicTypes.CURRENCY.ETH;
```

Comment:
Not 100% sure but: isn't it a good time to remove either the distinction between the 2 or at least to replace the ETH part with a more generic `NATIVE_TOKEN` ? Now that we have network, it makes sense, IMO. We will never have a CURRENCY.ETH on `network: bitcoin` or reversely and I'm not sure it ever helps to know that `BTC-bitcoin` is very close to `BTC-bitcoin-testnet`